pkg/monitor/worker.go

```diff
@@ -198,10 +236,12 @@ func (mon *monitor) workOne(ctx context.Context, log *logrus.Entry, doc *api.Ope
 		return
 	}
 
-	hiveRestConfig, err := mon.liveConfig.HiveRestConfig(ctx, 1)
-	if err != nil {
-		// TODO(hive): Update to fail once we have Hive everywhere in prod and dev
-		log.Info(err)
+	// once sharding is implemented, we will have the shard set on the api.OpenShiftClusterDocument
+	// e.g. shard := mon.hiveShardConfigs[doc.shardIndex]
+	shard := 1
+	hiveRestConfig, exists := mon.getHiveShardConfig(shard)
+	if !exists {
+		log.Errorf("no hiveShardConfigs set for shard %d", shard)
```

Comment:
IMO this should be a warning since we 'plan' on not having a config and already have an error log when we can't fetch it.

Reply:
The only place we plan on not having config is in FairFax. I've changed it, but we should change it back once we go live there.